src/app/controllers/followerships_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FollowershipsController < ApplicationController
+  before_action :set_user
+
+  def create
+    current_user.follow(@user)
+    redirect_back fallback_location: [@user, :functions]
+  end
+
+  def destroy
+    current_user.unfollow(@user)
```

Comment:
You can get the followership here in a before action like other controllers, this will allow you to raise `ActiveRecord::RecordNotFound` in case the record doesn't exist. and then you can destroy it here